apps/server/src/modules/school/service/school.service.spec.ts

```diff
@@ -301,6 +301,17 @@ describe('SchoolService', () => {
 				expect(resp.enableMigrationStart).toBeTruthy();
 			});
 		});
+		describe('when DO does not have oauthMigrationPossible and oauthMigrationFinished', () => {
```

Comment:
Please add empty lines between tests and in the test case between the logic chapters: arrange, act and assert